src/main/java/org/folio/entitlement/integration/kafka/KafkaEventUtils.java

```diff
@@ -91,4 +96,20 @@ public static PermissionMappingValue getPermissionValueMapping(String key) {
   public static boolean isPermissionMappingExist(String key) {
     return permissionMapping.containsKey(key);
   }
+
+  public static void addMissingResources(ModuleDescriptor newDescriptor) {
```

Comment:
Can this method be placed somewhere but not inside KafkaEventUtils, because it's not clear how it relates to Kafka at all